From 8b9e63efb7b578a9307b63d939ecdab7f3567383 Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date: Sun, 8 Jan 2017 20:49:27 +0000
Subject: [PATCH] fix build rom_mwskins_mod

---
 src/mame/drivers/atlantis.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mame/drivers/atlantis.cpp b/src/mame/drivers/atlantis.cpp
index 48d237a..9e2baa5 100644
--- a/src/mame/drivers/atlantis.cpp
+++ b/src/mame/drivers/atlantis.cpp
@@ -968,4 +968,5 @@ DRIVER_INIT_MEMBER(atlantis_state,mwskins)
 
 GAME( 2000, mwskins,    0,      mwskins, mwskins, atlantis_state,  mwskins,   ROT0, "Midway", "Skins Game (1.06)", MACHINE_NOT_WORKING | MACHINE_NO_SOUND )
 GAME( 2000, mwskinsa, mwskins,  mwskins, mwskins, atlantis_state,  mwskins,   ROT0, "Midway", "Skins Game (1.06, alt)", MACHINE_NOT_WORKING | MACHINE_NO_SOUND )
+GAME( 2000, mwskinsa_mod, mwskins,  mwskins, mwskins, atlantis_state,  mwskins,   ROT0, "Midway", "Skins Game (1.04, mod)", MACHINE_NOT_WORKING | MACHINE_NO_SOUND )
 GAME( 2000, mwskinso, mwskins,  mwskins, mwskins, atlantis_state,  mwskins,   ROT0, "Midway", "Skins Game (1.04)", MACHINE_NOT_WORKING | MACHINE_NO_SOUND )
-- 
2.9.0

