From 514898b9a10dfc503ce377ff89f5baa45e8c5654 Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <dude@zaplabs.com>
Date: Tue, 3 Nov 2015 08:53:15 -0600
Subject: [PATCH 08/12] fix windows bgfx link in sdl builds

Signed-off-by: Jeffrey Clark <dude@zaplabs.com>
---
 scripts/src/osd/sdl.lua | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/src/osd/sdl.lua b/scripts/src/osd/sdl.lua
index edac636..591b75d 100644
--- a/scripts/src/osd/sdl.lua
+++ b/scripts/src/osd/sdl.lua
@@ -99,6 +99,11 @@ function maintargetosdoptions(_target,_subtarget)
 
 	configuration { "mingw*" or "vs*" }
 		targetprefix "sdl"
+		if USE_BGFX == 1 then
+			links {
+				"psapi"
+			}
+		end
 
 	configuration { }
 end
-- 
2.4.3

