From 140dee0e2b5490bae1c6dba36616bda77f635d72 Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <dude@zaplabs.com>
Date: Sun, 3 Jan 2016 20:47:17 -0600
Subject: [PATCH 10/12] set plist dependency before task to fix parallel build

Signed-off-by: Jeffrey Clark <dude@zaplabs.com>
---
 scripts/src/main.lua | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/scripts/src/main.lua b/scripts/src/main.lua
index 9d0d9a9..0e8c6c8 100644
--- a/scripts/src/main.lua
+++ b/scripts/src/main.lua
@@ -201,16 +201,15 @@ end
 	end
 
 	if _OPTIONS["targetos"]=="macosx" and (not override_resources) then
+		dependency {
+			{ "$(TARGET)" ,  GEN_DIR  .. "resource/" .. _subtarget .. "-Info.plist", true  },
+		}
 		linkoptions {
 			"-sectcreate __TEXT __info_plist " .. _MAKE.esc(GEN_DIR) .. "resource/" .. _subtarget .. "-Info.plist"
 		}
 		custombuildtask {
 			{ MAME_DIR .. "src/version.cpp" ,  GEN_DIR .. "resource/" .. _subtarget .. "-Info.plist",    {  MAME_DIR .. "scripts/build/verinfo.py" }, {"@echo Emitting " .. _subtarget .. "-Info.plist" .. "...",    PYTHON .. " $(1)  -p -b " .. _subtarget .. " $(<) > $(@)" }},
 		}
-		dependency {
-			{ "$(TARGET)" ,  GEN_DIR  .. "resource/" .. _subtarget .. "-Info.plist", true  },
-		}
-
 	end
 	local rctarget = _subtarget
 
-- 
2.4.3

