From 302b4e51fd70abf895c76bb4022371b9ed0f06e5 Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <dude@zaplabs.com>
Date: Mon, 4 Jan 2016 21:41:56 -0600
Subject: [PATCH 12/12] osx clang, silence flac unknown attribute error

---
 scripts/src/3rdparty.lua | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/src/3rdparty.lua b/scripts/src/3rdparty.lua
index c4f460f..75ecdd2 100644
--- a/scripts/src/3rdparty.lua
+++ b/scripts/src/3rdparty.lua
@@ -275,6 +275,11 @@ end
 		buildoptions {
 			"-Wno-enum-conversion",
 		}
+		if _OPTIONS["targetos"]=="macosx" then
+			buildoptions_c {
+				"-Wno-unknown-attributes",
+			}
+		end
 	end
 	configuration { }
 
-- 
2.4.3

