From 4a471d5eacf5a45a8abcbaac79746569c18ca1bf Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <dude@zaplabs.com>
Date: Mon, 4 Jan 2016 21:41:56 -0600
Subject: [PATCH 12/13] osx clang, silence flac unknown attribute error

Signed-off-by: Jeffrey Clark <dude@zaplabs.com>
---
 scripts/src/3rdparty.lua | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/src/3rdparty.lua b/scripts/src/3rdparty.lua
index 16b86dc..d66899b 100644
--- a/scripts/src/3rdparty.lua
+++ b/scripts/src/3rdparty.lua
@@ -280,6 +280,11 @@ end
 		buildoptions {
 			"-Wno-enum-conversion",
 		}
+		if _OPTIONS["targetos"]=="macosx" then
+			buildoptions_c {
+				"-Wno-unknown-attributes",
+			}
+		end
 	end
 	configuration { }
 
-- 
2.4.3

