From 2502a497c7c3d3c2740fdbf04a8fdda37ca736cd Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date: Wed, 29 Jun 2016 17:31:40 +0000
Subject: [PATCH 2/3] android build folders (nw)

Signed-off-by: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
---
 scripts/toolchain.lua | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/scripts/toolchain.lua b/scripts/toolchain.lua
index 9bfa81f..e85369b 100644
--- a/scripts/toolchain.lua
+++ b/scripts/toolchain.lua
@@ -772,9 +772,15 @@ function toolchain(_buildDir, _subDir)
 	configuration { "openbsd", "x64", "Debug" }
 		targetdir (_buildDir .. "openbsd" .. "/bin/x64/Debug")
 
+	configuration { "android-*", "Release" }
+		targetdir (_buildDir .. "android/bin/" .. _OPTIONS["PLATFORM"] .. "/Release")
+		objdir (_buildDir .. "android/obj/" .. _OPTIONS["PLATFORM"] .. "/Release")
+
+	configuration { "android-*", "Debug" }
+		targetdir (_buildDir .. "android/bin/" .. _OPTIONS["PLATFORM"] .. "/Debug")
+		objdir (_buildDir .. "android/obj/" .. _OPTIONS["PLATFORM"] .. "/Debug")
+
 	configuration { "android-*" }
-		targetdir (_buildDir .. "android-" .. _OPTIONS["PLATFORM"] .. "/bin")
-		objdir (_buildDir .. "android-" .. _OPTIONS["PLATFORM"] .. "/obj")
 		includedirs {
 			MAME_DIR .. "3rdparty/bgfx/3rdparty/khronos",
 			"$(ANDROID_NDK_ROOT)/sources/cxx-stl/llvm-libc++/libcxx/include",
-- 
2.9.0

